gfauto/gfauto/process_seeds.py

```diff
@@ -50,16 +51,33 @@ def process_chunk(  # pylint: disable=too-many-locals;
             end_line = 0
             found_bug = False
             for i, line in enumerate(lines):
-                match = re.fullmatch(r"STATUS (\w+)\n", line)
-                if not match:
-                    continue
-                status = match.group(1)
-                if status == "SUCCESS":
-                    start_line = i + 1
-                    continue
-                found_bug = True
-                end_line = i + 1
-                break
+                if look_for_spirv_opt_errors:
+                    match = re.match(r"Command failed: \['/usr/bin/catchsegv', .*spirv-opt'", line)
+                    if match:
+                        found_bug = True
+                        start_line = i + 1
+                        # TODO: Is there a reliable way to find the end of a spirv-opt failure?
```

Comment:
Maybe:
 - Don't break once a bug is found.
 - if found_bug, do something different: 
    - Just look for "Exec (verbose): ..." to find the next "reduce" command, set the end line based on this and then break.
   - if we find "gfauto_interestingness_test: finished ..." then the output above this is actually the output from part of a reduction, so something has gone wrong.
   - if we don't find anything else and get to the end of the file, this is also fine. It just means we did not do a reduction and the file immediately ends after the "Command failed:" output.